pacman/model/graphs/graph.py

```diff
@@ -56,28 +50,26 @@ class Graph(ConstrainedObject, AbstractGraph):
         # map between labels and vertex
         "_vertex_by_label",
         # count of vertex which had a None or already used label
-        "_none_labelled_vertex_count",
-    ]
+        "_unlabelled_vertex_count"]
 
-    def __init__(self, allowed_vertex_types, allowed_edge_types,
-                 allowed_partition_types, label):
+    def __init__(self, allowed_vertex_types, allowed_edge_types, label):
         """
-        :param allowed_vertex_types:\
+        :param allowed_vertex_types:
             A single or tuple of types of vertex to be allowed in the graph
-        :param allowed_edge_types:\
+        :type allowed_vertex_types: type or tuple(type, ...)
+        :param allowed_edge_types:
             A single or tuple of types of edges to be allowed in the graph
-        :param allowed_partition_types:\
-            A single or tuple of types of partitions to be allowed in the graph
+        :type allowed_edge_types: type or tuple(type, ...)
         :param label: The label on the graph, or None
+        :type label: str or None
         """
         super(Graph, self).__init__(None)
         self._allowed_vertex_types = allowed_vertex_types
         self._allowed_edge_types = allowed_edge_types
-        self._allowed_partition_types = allowed_partition_types
```

Comment:
While this change makes sense in the current stack it does not when considering the future SDRAM edges stuff.

  Will revert in the Cherry pick

Reply:
I think I put this back in.